docs/docs/30-administration/15-agent-config.md

```diff
@@ -132,3 +132,26 @@ See [Docker backend configuration](backends/docker/#configuration)
 ### `WOODPECKER_BACKEND_SSH_*`
 
 See [SSH backend configuration](backends/ssh/#configuration)
+
+
+## Image Cleanup
```

Comment:
@hay-kot Not sure if this is the best place to put it. I would move it to the docker backend page as it is only valid / needed for that backend.

Reply:
Should be good now 👍